msos/NativeMethods.cs

```diff
@@ -1,22 +1,171 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Runtime.InteropServices;
-using System.Text;
-using System.Threading.Tasks;
+using Microsoft.Win32.SafeHandles;
+using FileTime = System.Runtime.InteropServices.ComTypes.FILETIME;
+using static msos.NativeStructs;
 
 namespace msos
 {
-    using FileTime = System.Runtime.InteropServices.ComTypes.FILETIME;
-
     static class NativeMethods
     {
+        #region DbgHelp
+        //public const int ERR_ELEMENT_NOT_FOUND = 1168;
+
+        [DllImport("Dbghelp.dll", SetLastError = true)]
+        public static extern bool MiniDumpReadDumpStream(IntPtr BaseOfDump,
```

Comment:
Do we still need this function here, even though you changed `DumpReader` in our CLRMD fork?